README.md

```diff
@@ -1,114 +1,85 @@
 # OpenFisca Legislation explorer
 
-[![CircleCI](https://circleci.com/gh/openfisca/legislation-explorer.svg?style=svg)](https://circleci.com/gh/openfisca/legislation-explorer)
+A web user interface to explore an [OpenFisca](http://openfisca.org/) tax and benefit system model.
```

Comment:
I voluntarily started stepping away from that phrasing due to ongoing discussions around whether these models are for “countries” or “regions” or “towns”…

Reply:
I think the `Country package` terminology is used throughout the documentation and it is less confusing than `tax and benefit system model`.
I am sure the ongoing dicussions come up with a good alternative. :)

Reply:
The expression `tax and benefit system` is also used a lot in the doc, and is more semantic than `country package`, which is more technical. We don't usually append the word `model` though.

The concepts of Country Package and Tax Benefit System are indeed close and often used without distinction. Historically, the difference was:
- The TBS is what contains variables, parameters, entities. It represents the legislation.
- A country package is just a python package that wraps the TBS so that it can be installed with `pip`. It can also contains a few reforms that are not technically part of the TBS, but that's a detail.

So I'd be in favor of:

> A web user interface to explore an [OpenFisca](http://openfisca.org/) tax and benefit system.